content/docs/guide/map_submission/fixing_entities.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Fixing Entities
+categories:
+  - guide
+tags:
+  - mapping
+  - guidelines
+weight: 4
+---
+
+# Introduction
+This guide provides a list of fixes to apply to entities when porting maps.  
+If the entity you are looking for is not in here please ask for help in **#map-porting** channel on our [Discord](https://discord.gg/momentummod)  
+Detailed information on entities can also be found on [Valve Developer Wiki](https://developer.valvesoftware.com/wiki/Main_Page)
+
+
+{{< hint info >}} 
+
+This guide will teach you how to make changes using 'Entity Tools' and [export them to Lumper](/guide/map_submission/fixing_entities/#export-to-lumper)  
+When making changes using 'Entity Tools' it's perfectly fine to export them to Lumper only once, when you are done with everything, however there is no downside to gradually saving your progress
+
+{{< /hint >}}
+
+# Boosters
+Boosting players can be achieved in various ways and therefore triggers need to be evaluated on a case-by-case basis.  
+This guide will teach you how to fix every common scenario
+
+## trigger_push
+This trigger applies a constant boost to a player inside it's volume. Consider 2 following scenarios
+
+### Scenario 1: Constant push is required
+On some maps it is necessary for the player to be constantly pushed while inside of the trigger.  
+In that case no modifications are necessary!
+
+### Scenario 2: Only a single push matters
+In this case modifications are required depending on where the trigger is placed.
+
+1. Open entity tools by typing `devui_show entitytools` in console
+    - You can bind this to a key for ease of access, `bind <key> "devui_show entitytools"` in console
+2. Find the trigger you want to edit (TODO: ELABORATE WHEN ENTITY TOOLS ARE BETTER, RIO IS FIXING THEM)
+
+#### Trigger is in the middle of the map/stage:
+
+3. Check the 'cooldown' box and type **1** in the textbox
+4. Click 'Apply Changes`
+5. Redo steps 2-4 for all appropriate triggers
+6. [Export to Lumper](/guide/map_submission/fixing_entities/#export-to-lumper)
+![Surf Ramp Boost](/images/map_porting/surf_ramp_boost.png)
+
+#### Trigger is at the start of the map/stage:
+3. Fail the map/stage and don't move your mouse so you look directly at the trigger
+    - You can also set your angle by using `setang X Y Z` command in console
+    - If setting the angle manually use multiples of 90 such as `setang 180 0 0` or `setang 0 90 0` to orient yourself properly
+4. Walk into the trigger by pressing **W only**
+    - The game will automatically get all relevant information after using the trigger in this way
+5. If the booster is on completely flat ground ( no slopes next to it ) check 'Standable Surface Collision Filter'
```

Comment:
I don't think you need the surface filter for setspeed triggers. This is only used to convert trigger_push boosts to conveyors (ex: surf_fruits apple stage).

Reply:
Also your guide is missing the scenarios for converting to Standable Surface Collision Filter and OnJump triggers. I think my guide originally had these.